client/app/student/dashboard/dashboard.html

```diff
@@ -6,18 +6,18 @@
         </div>
         <div class="col-sm-9 dashboard">
           <h1 class="page-header">Dashboard</h1>
-          <h2> Events </h2>
+          <h2> Event Assignements </h2>
```

Comment:
If a student were to have two classes that both required an assignment at the same event, they would see what looks like a duplicate event. With the Event Assignments heading, if we were to mandate instructors to have event assignment titles, it will make this more clear. @kmcnellis does that make sense?